deploy/layered-sre-authorization/02-layered-cs-sre-admin.SubjectPermission.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: layered-cs-sre-admin
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - layered-cs-sre-admin
+  permissions:
+  - allowFirst: true
+    clusterRoleName: dedicated-admins-cluster
```

```suggestion
    clusterRoleName: layered-cs-sre-admin
```
Suspect this is a copy/paste error, intent is to authorize with the LP admin ClusterRole